AutomatedTesting/Gem/PythonTests/Prefab/TestSuite_Main_Optimized.py

```diff
@@ -80,18 +80,24 @@ class test_SC_Spawnables_NestedSpawn(EditorSharedTest):
 
     class test_SC_Spawnables_DespawnOnEntityDeactivate(EditorSharedTest):
         from .tests.spawnables import SC_Spawnables_DespawnOnEntityDeactivate as test_module
-        
+
+# Temporarily skipped because this fails on linux
+"""
```

Comment:
We should utilize the pytest.mark.skip option for these instead of commenting out the test. Can even skip it only on Linux if that's where it's failing by doing something like this:
```
@pytest.mark.skipif(sys.platform.startswith("linux"), reason="<Link bug for Linux failure here>")
```